components/toggl/actions/create-project/create-project.mjs

```diff
@@ -0,0 +1,138 @@
+import toggl from "../../toggl.app.mjs";
+
+export default {
+    name: "Create a new Project",
+    version: "0.0.1",
+    description: "Create project for given workspace. [See docs here.](https://developers.track.toggl.com/docs/api/projects#post-workspaceprojects)",
+    type: "action",
+    key: "toggle-create-new-project",
+    props: {
+        toggl,
+        name: {
+            type: "string",
+            label: "Name",
+            description: 'unique for client and workspace'
+        },
+        workspace_id: {
```

Comment:
If this is the same as the prop in the previous action, they should be a propDefinition in the app file to be used by both
```suggestion
        workspaceId: {
```